docs/src/components/MDXContent/MDXContent.scss

```diff
@@ -0,0 +1,113 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+@mixin heading-style($spacing) {
+  margin-bottom: $spacing;
+
+  &:not(:first-of-type) {
+    margin-top: $spacing;
+
+    @include bpk-breakpoint-mobile {
+      margin-top: 0;
+    }
+  }
+}
+
+.bpkdocs-mdx-page {
+  &__content {
+    padding: $bpk-spacing-lg * 2;
+    background-color: $bpk-color-sky-gray-tint-07;
+
+    @include bpk-breakpoint-mobile {
+      padding: $bpk-spacing-lg;
+    }
+  }
+
+  &__table {
+    // Using `!important` as we need to override the baremetal HTML styling, which takes precedence.
+    /* stylelint-disable declaration-no-important */
+    width: auto !important;
+    // Applying a custom border to the table as the `bpk-border` mixin using box-shadow has side-effects
+    // when used inside a scroll container. This is due to the box-shadow not being measured as part of the
+    // width of the child element.
+    /* stylelint-disable-next-line unit-blacklist */
+    border: solid 1px $bpk-color-sky-gray-tint-04;
+    box-shadow: none !important;
+    /* stylelint-enable */
+  }
+
+  &__mobile-scroll-container {
+    margin-bottom: $bpk-spacing-sm;
+  }
+
+  &__mobile-scroll-indicator {
+    &--trailing {
+      &--dark {
```

Comment:
We can remove the `--dark` and `--light` parts and just use the light styling everywhere, as the `darkBackground` prop isn't used.

Reply:
See above